server/lib/statsd-collector.js

```diff
@@ -101,7 +109,7 @@ function sendMarketingImpressions(context, marketing, tags) {
 function sendNavigationTiming(context, navigationTiming, tags) {
   if (navigationTiming) {
     for (var key in navigationTiming) {
-      if (typeof navigationTiming[key] === 'number') {
+      if (isEventOffsetValid(navigationTiming[key], NAVIGATION_TIMING_MAX_OFFSET)) {
```

Comment:
One of the questions I had here is if the value is > NAVIGATION_TIMING_MAX_OFFSET, should we filter the value, or set the value to the max?